bootstrap-salt.sh

```diff
@@ -2576,6 +2591,7 @@ __install_saltstack_ubuntu_repository() {
 
     __apt_key_fetch "$SALTSTACK_UBUNTU_URL/SALTSTACK-GPG-KEY.pub" || return 1
 
+    __wait_for_apt
     apt-get update
```

Comment:
Maybe we could wrap `apt-get update` command with waiter in an "internal" function (like `__apt_get_install_noinput`) to keep the code DRY?

Reply:
I thought of that, but we're calling bare `apt-get update` in many places. I thought this way would be better until we have time to do a larger refactor. I don't have time to do a larger change at this time, along with the requisite testing. Is that something you'd be interested in?

Reply:
No worries, probably I could do that later.